app/src/components/v-upload.vue

```diff
@@ -24,17 +31,30 @@ const props = withDefaults(defineProps<Props>(), {
 	fromUser: true,
 });
 
-const emit = defineEmits(['input']);
+const emit = defineEmits<{
+	input: [files: null | File | File[]];
+	start: [controller: UploadController];
+}>();
 
 const { t } = useI18n();
 
+let uploader: tus.Upload | null = null;
+
 const { uploading, progress, upload, onBrowseSelect, done, numberOfFiles } = useUpload();
 const { onDragEnter, onDragLeave, onDrop, dragging } = useDragging();
 const { url, isValidURL, loading: urlLoading, importFromURL } = useURLImport();
 const { setSelection } = useSelection();
 const activeDialog = ref<'choose' | 'url' | null>(null);
 const input = ref<HTMLInputElement>();
 
+function abortUpload() {
+	uploader?.abort();
+}
+
+defineExpose({
+	abort: abortUpload,
+});
```

Comment:
Never seen this before. When do we rely on `abort` being exposed to the parent and why is it needed?